src/libprojectM/Renderer/hlslparser/src/Engine.cpp

```diff
@@ -59,8 +64,31 @@ bool String_EqualNoCase(const char * a, const char * b) {
 #endif
 }
 
+static inline double iss_strtod(const char * in, char ** end) {
```

Comment:
nit: let the compiler decide for itself if it wants to inline this or not: is my advice.

alternativelhy you can use an anonymous namespace instead of static

```namespace  {
// what would be a static before
} // anonymous namespace
```

Reply:
Well, inline is a suggestion to the compiler. It doesn't force it to do anything. The static was there to make inlining it easier.
I do agree though. It was only done like that because it's used in that specific place.

If you think it's better, I can make it a normal function that's part of the class, private or public.